packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2636,6 +2657,66 @@ void main() {
     lastOffset = tester.getTopLeft(find.text('11'));
     expect(tester.getTopLeft(find.text('2022')).dy, lastOffset.dy);
   });
+
+  group('CupertinoTimerPicker hour label animation', () {
+    testWidgets('animates from plural to singular when scrolling from 2 to 1', (
```

Comment:
It seems to me that these tests would also pass if the text is not animated. If so, can you think of a way to test it? Maybe find the `FadeTransition` / `RenderAnimatedOpacity` around the `s` text and verify its `opacity`?